cli/run_util.go

```diff
@@ -580,6 +580,57 @@ func activateStepLibStep(stepIDData models.StepIDData, destination, stepYMLCopyP
 	return info, didStepLibUpdate, nil
 }
 
+func expandStepInputs(
+	inputs []envmanModels.EnvironmentItemModel,
+	environments []envmanModels.EnvironmentItemModel,
+) map[string]string {
+	stepInputs := make(map[string]string)
+
+	// Retrieve all non-sensitive input values
+	for _, environmentItem := range inputs {
+		if err := environmentItem.FillMissingDefaults(); err != nil {
+			log.Warnf("Failed to fill missing defaults, skipping input")
+			continue
+		}
+
+		options, err := environmentItem.GetOptions()
+		if err == nil && *options.IsSensitive == false {
+			if inputName, inputValue, err := environmentItem.GetKeyValuePair(); err == nil {
+				inputString := fmt.Sprintf("%v", inputValue)
```

Comment:
Why is this line needed? Also used at line 613.

Reply:
this is a string conversion from interface{}

Reply:
was string already, changed